Mac/BuildScript/openssl-mac-arm64.patch

```diff
@@ -0,0 +1,41 @@
+diff -ur openssl-1.1.1g-orig/Configurations/10-main.conf openssl-1.1.1g/Configurations/10-main.conf
```

Comment:
I've checked the OpenSSL repo and it does not yet contain a similar patch in the 1.1.1 branch. This patch is still needed.